external/kafka-0-10/src/main/scala/org/apache/spark/streaming/kafka010/KafkaUtils.scala

```diff
@@ -177,3 +182,172 @@ object KafkaUtils extends Logging {
     }
   }
 }
+
+private[kafka010] class KafkaUtilsPythonHelper extends Logging {
+  import KafkaUtilsPythonHelper._
+
+  def createDirectStream(
+      jssc: JavaStreamingContext,
+      locationStrategy: LocationStrategy,
+      consumerStrategy: ConsumerStrategy[Array[Byte], Array[Byte]]
+    ): JavaDStream[Array[Byte]] = {
+    fixKafkaParams(consumerStrategy.executorKafkaParams)
```

Comment:
Good point.  I think the reasonable choices here are to wrap the passed in consumer strategy with a python-specific class that delegates and does the defensive copy, or to just error if the serializers are not correct.  I think I'm more in favor of just throwing an error, having to specify the correct serializers isn't any more work than the scala / java api, and it's more explicit.